products/meilisearch.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Meilisearch
+category: server-app
+tags: meilisearch
+iconSlug: meilisearch
+permalink: /meilisearch
+versionCommand: meilisearch --version
+releasePolicyLink: https://www.meilisearch.com/docs/learn/update_and_migration/versioning
+changelogTemplate: https://github.com/meilisearch/meilisearch/releases
+releaseDateColumn: true
+
+identifiers:
+-   repology: meilisearch
+
+auto:
+-   git: https://github.com/meilisearch/meilisearch.git
+
+releases:
+-   releaseCycle: "1.5"
+    releaseDate: 2023-11-20
+    eol: false
+    latest: "1.5.1"
+    latestReleaseDate: 2023-12-13
+
+---
+
+> [Meilisearch](https://www.meilisearch.com/) is a RESTful search API. It aims to be a ready-to-go solution for everyone who wants a fast and relevant search experience for their end-users.
+
+Meilisearch releases and their associated binaries are available [in this GitHub page](https://github.com/meilisearch/meilisearch/releases).
+
+The binaries are versioned following [SemVer conventions](https://semver.org/). To know more, read our [versioning policy](https://github.com/meilisearch/engine-team/blob/main/resources/versioning-policy.md).
+
```

Comment:
From what I read on https://www.meilisearch.com/docs/learn/update_and_migration/versioning, it would be good to also:

- add a sentence to says that [Meilisearch engine version numbers have no relationship to SDK version numbers](https://www.meilisearch.com/docs/learn/update_and_migration/versioning#sdk-versioning),
- and maybe talk about the cloud offer ?

Reply:
I have added two sentences about the SDK and the cloud